cmake/Modules/LibAddHaskellPlugin.cmake

```diff
@@ -233,28 +254,6 @@ macro (add_haskell_plugin target)
 			remove_plugin (${target} "${PLUGINPROCESS_NOTFOUND_INFO}, but required for haskell plugins")
 		endif (PLUGINPROCESS_FOUND)
 	endif (DEPENDENCY_PHASE)
-
-	# compile our c wrapper which takes care of invoking the haskell runtime
-	# the actual haskell plugin gets linked in dynamically as a library
-	add_plugin (${target}
```

Comment:
add_plugin needs to be present at all stages, to disable plugins remove_plugin needs to be called.

Reply:
that was not the problem